src/site/content/en/blog/preconnect-and-dns-prefetch/index.md

```diff
@@ -146,5 +148,14 @@ Implementing `dns-prefetch` fallback in the same `<link>` tag causes a bug in Sa
 
 {% endCompare %}
 
+## Effect on Largest Contentful Paint (LCP)
+
+The goal of `dns-prefetch` and `preconnect` is to lessen the amount of time it takes to connect to another origin. The ultimate result is that the time to discover a resource loaded from another origin should be reduced, even if only somewhat.
+
+Where [Largest Contentful Paint (LCP)](/lcp/) is concerned, the least you can do to load a cross-origin resource faster is to use `preconnect`&mdash;but a well-placed [`preload`](/preload-critical-assets/) would most likely be better, as an [LCP candidate](/lcp/#what-elements-are-considered) is a crucial part of the user experience. That said, a `preconnect` can only help, even if not as much as a `preload`.
+
+All that said, `preconnect` is less expensive than `preload` in terms of bandwidth usage. While multiple `preload` hints can cause bandwidth contention, the same is less true for `preconnect`. Use it to reduce resource discovery time, which can have a positive effect on LCP. 
```

Comment:
"That said" in last paragraph. "All that said" in this paragraph. Can we reword to avoid the repetition.

And it's true that `preconnect` is less expensive, but still can use up a bit of bandwith with TLS certificates - especially at this critical stage of the initial load of the page. We've seen some sites use a quite-frankly [ridiculous 1286 preconnects](https://docs.google.com/spreadsheets/d/1Mw6TjkIClRtlZPHbij5corOZbaSUp-vgTVq3Ig18IwQ/edit#gid=547570509) which surely has an impact on bandwidth.

So maybe tone down this sentence or add a warning?